packages/crowbar/crowbar.0.2/opam

```diff
@@ -0,0 +1,40 @@
+opam-version: "2.0"
+maintainer: "stephen.dolan@cl.cam.ac.uk"
+homepage: "https://github.com/stedolan/crowbar"
+bug-reports: "https://github.com/stedolan/crowbar/issues"
+dev-repo: "git+https://github.com/stedolan/crowbar.git"
+license: "MIT"
+build: [
+  [ "dune" "build" "-p" name "-j" jobs ]
+]
+run-test: [
+  [ "dune" "build" "-p" name "-j" jobs
+    "@examples/calendar/runtest"
+    "@examples/fpath/runtest"
+    "@examples/map/runtest"
+    "@examples/serializer/runtest"
+    "@examples/uunf/runtest"
+  ]
+]
+depends: [
+  "dune" {build & >= "1.1"}
```

Comment:
Could you remove the `build` flag here? Dune is not compatible with this flag and I can't edit your PR.